examples/android/CHIPTool/app/src/main/AndroidManifest.xml

```diff
@@ -17,8 +17,11 @@
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
     <uses-permission android:name="android.permission.NFC" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.MANAGE_EXTERNAL_STORAGE" />
```

Comment:
it is dangerous permission，should be replaced by SAF

Reply:
SAF looks better here, thanks

Reply:
I agree with this comment. However, this part does not seem to be an important point in OTA implementation. After merging, how about modifying it using SAF?

Reply:
I tried using SAF, but I am having a problem with the file not being read.

Of course, this permission is a dangerous permission, but I think this permission exists only for testing. In the case of actual commercial apps, I think this permission is not necessary because the file is downloaded from DCL and only the storage within the app is used. Is there really a need to delete this?

Reply:
I found that how to use SAF. I modified code and removed dangerous permission.